### PR TITLE
Creation of CustomElements array and corresponding elements. 

### PR DESCRIPTION
**Pull request by kevinkirkl3y**

### CODE REVIEW

**Comment by kevinkirkl3y** [comment]
> I've created a [PR](https://github.com/kevinkirkl3y/Orange-Button-Taxonomy/pull/1) to your branch adding `CustomElements` to `System`. Please, take a look into it before merging this PR

Would you be willing to meet with the work group to discuss your use-case? The meeting is on Fridays, 2:30 pm PST.

**Comment by naulacambra** [comment]
@kevinkirkl3y I would love to. How I could meet with the work group?

**Review by jrippingale — APPROVED** [review]

**Review by jrippingale — APPROVED** [review]

**Comment by cwhanse** [comment]
@kevinkirkl3y I think you need to merge upstream, this PR would undo changes from #228

**Comment by kevinkirkl3y** [comment]
@cwhanse I ran into some issues during the merge process but it should be resolved.

File: Master-OB-OpenAPI.json [hunk 1]
```diff
@@ -18075,6 +18228,19 @@
         }
       }
     },
+    "BLOBTypeItemType": {
+      "description": "A BLOB (binary large object) is a varying-length binary string that can be up to 2,147,483,647 characters long. Like other binary types, BLOB strings are not associated with a code page. In addition, BLOB strings do not hold character data.",
```
Comment (cwhanse) [inline Master-OB-OpenAPI.json:18232]:
```suggestion
      "description": "A BLOB (binary large object) is a varying-length string that encodes binary data. Do not use BLOB for character data, use StringTypeItemType instead.",
```

**Review by cwhanse — COMMENTED** [review]
@kevinkirkl3y can you check these suggested edits? IBM blobs can be much larger than 2GB. 

Please add an entry to whatsnew for BlobTypeItemType

**Review by cwhanse — APPROVED** [review]

**Pull request merged**